nucleus/common/common-util/src/main/java/com/sun/common/util/logging/LoggingOutputStream.java

```diff
@@ -118,7 +118,9 @@ public void flush() throws IOException {
         GFLogRecord logRecordWrapper = new GFLogRecord(logRecord);
         logRecordWrapper.setThreadName(Thread.currentThread().getName());
 
-        pendingRecords.offer(logRecordWrapper);
+        if (!pendingRecords.offer(logRecordWrapper)) {
+            System.out.println("Queue is full");
```

Comment:
This should use the logger, shouldn't it?

Reply:
It should indeed